articles/iot-edge/how-to-retrieve-iot-edge-logs.md

```diff
@@ -155,6 +157,8 @@ Use the **UploadModuleLogs** direct method to send the requested logs to a speci
 ::: moniker range=">=iotedge-2020-11"
 
 > [!NOTE]
+> Use the `since` and `until` filter options to limit the amount of logs retrieved. Calling this direct method without bounds will retrieve all the logs which may be large, time consuming or costly.
```

Comment:
```suggestion
> Use the `since` and `until` filter options to limit the range of logs retrieved. Calling this direct method without bounds retrieves all the logs which may be large, time consuming, or costly.
```